src/SDKs/Azure.Base/data-plane/Azure.Base/Azure.Base.csproj

```diff
@@ -26,4 +26,8 @@
     <PackageReference Include="System.Numerics.Vectors" Version="4.5.0" />
     <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.5.2" />
   </ItemGroup>
+
+  <ItemGroup>
+    <InternalsVisibleTo Include="Azure.Base.Tests" />
```

Comment:
Why do we need this?

Reply:
HttpPipelineEventSource is now internal and used in test.

Reply:
Oh wait, my bad. That's my failed attempt at puitting attribute into csproj. Will revert.